### ISSUE TITLE
This example kills vw

### ISSUE THREAD

**Issue by ekg** [ekg/hhga#20]
[kills-vw.txt](https://github.com/ekg/hhga/files/234648/kills-vw.txt)

I am wondering if it is too large (1.8M).

What happens:

```
-> % cat kills-vw.txt | vw --ect 7                                       
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = 
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
warning:  is not a good int, replacing with 0
warning: iproper:0 is not a good float, replacing with 0
terminate called after throwing an instance of 'VW::vw_exception'
  what():  label 0 is not allowed for multiclass.  Valid labels are {1,k}
[1]    879 done                 cat kills-vw.txt | 
       881 abort (core dumped)  vw --ect 7
```

**Comment by nikete** [comment]
perhaps related:

> name=lodz; time vw -d robots134h.shuf.hhga.gz --binary  --passes 40 --ect 7 -q ss -q mm  --save_per_pass --cache_file $name.cache -f ~/$name.model --save_resume -b 25  2>&1  | tee ~/$name.log  
> creating quadratic features for pairs: ss mm 
> final_regressor = /home/freebayes/lodz.model
> Num weight bits = 25
> learning rate = 0.5
> initial_t = 0
> power_t = 0.5
> decay_learning_rate = 1
> creating cache_file = lodz.cache
> Reading datafile = robots134h.shuf.hhga.gz
> num sources = 1  
> average  since         example        example  current  current  current
> loss     last          counter         weight    label  predict features
> 1.000000 1.000000            1            1.0        2        1     4455
> 1.000000 1.000000            2            2.0        4        2     2690
> 0.750000 0.500000            4            4.0        2        2     3824
> 0.625000 0.500000            8            8.0        2        1     2184
> 0.500000 0.375000           16           16.0        2        1     1850
> 0.343750 0.187500           32           32.0        4        4     2909
> 0.234375 0.125000           64           64.0        2        2     2815
> 0.140625 0.046875          128          128.0        2        2     2488
> 0.125000 0.109375          256          256.0        2        2     2970
> 0.076172 0.027344          512          512.0        1        1     4472
> 0.054688 0.033203         1024         1024.0        4        4     2391
> 0.045410 0.036133         2048         2048.0        2        2     3277
> 0.036377 0.027344         4096         4096.0        2        2     2950
> 0.031128 0.025879         8192         8192.0        2        2     5475
> 0.026733 0.022339        16384        16384.0        2        2     2595
> 0.023193 0.019653        32768        32768.0        4        4     2037
> 0.020050 0.016907        65536        65536.0        2        2     2730
> 0.018036 0.016022       131072       131072.0        2        2     2548
> 0.016254 0.014473       262144       262144.0        2        2     2186
> 0.015036 0.013817       524288       524288.0        2        2     2347
> 0.013927 0.012817      1048576      1048576.0        2        2     3233
> 0.013055 0.012184      2097152      2097152.0        4        4     3319
> 0.012227 0.011399      4194304      4194304.0        4        4     1999
> 0.011593 0.010960      8388608      8388608.0        2        2     2660
> truncated example! wanted: 17319326526688563113 bytes
> terminate called after throwing an instance of 'VW::vw_exception'
>   what():  argh, a bug in caching of some sort!
> zsh: abort (core dumped)  vw -d robots134h.shuf.hhga.gz --binary --passes 40 --ect 7 -q ss -q mm    -f  | 
> zsh: done                 tee ~/$name.log
> vw -d robots134h.shuf.hhga.gz --binary --passes 40 --ect 7 -q ss -q mm    -f   5067.55s user 35.87s system 131% cpu 1:04:46.12 total
> tee ~/$name.log  0.00s user 0.00s system 0% cpu 1:04:46.11 total
> freebayes@ip-10-0-0-6 ~ %  what():  argh, a bug in caching of some sort!
